.github/workflows/script-evaluation-test.yml

```diff
@@ -0,0 +1,37 @@
+# A nightly job which downloads script evaluation dumps from S3 and runs a regression test.
+name: Script Evaluation Test
+on:
+  schedule:
+    - cron: 30 3 * * * # 3:30am every day 
+
+jobs: 
+  script-evaluation-test: 
+    runs-on: [self-hosted, plutus-benchmark]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.3.0
+
+      - name: Download and Unzip Dump Files
+        run: |
+          export LOCAL_DIR="$HOME/mainnet-script-dump-downloaded" 
+          nix develop --no-warn-dirty --accept-flake-config --command \
+            bash ./scripts/s3-sync-unzip.sh s3://plutus/mainnet-script-dump/ \*.event.bz2
+        env: 
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_DEFAULT_REGION: us-east-1
+          AWS_ENDPOINT_URL: https://s3.devx.iog.io
+
+      - name: Run
+        # Run the test cases sequentially. This ensures we don't need to simultaneously store
+        # multiple `ScriptEvaluationEvents`, which are large, in memory. Each test case
+        # contains many script evaluation events, and those are run in parallel based on
+        # the number of available processors.
+        run: |
+          export EVENT_DUMP_DIR="$HOME/mainnet-script-dump-downloaded"
+          nix develop --no-warn-dirty --accept-flake-config --command cabal update 
+          nix develop --no-warn-dirty --accept-flake-config --command \
```

Comment:
We need two calls to nix develop unfortunately, this won't work:
```
nix develop --no-warn-dirty --accept-flake-config --command cabal update \ 
  cabal update && cabal v2-run ... 
``` 
As 
`nix develop --command` won't allow this syntax -> it breaks at `&&`

Reply:
Alternatively, what about
```
nix run .#x86_64-linux.plutus.library.plutus-project-924.hsPkgs.plutus-ledger-api.components.exes.evaluation-test -- --num-threads=1
```